packages/store-ui/src/atoms/Button/Button.tsx

```diff
@@ -0,0 +1,12 @@
+import type { FC, ReactNode } from 'react'
```

Comment:
What do you think about changing the filename to index?

Reply:
I'll create an index file, thanks!